Multisig/UI/Transaction/TransactionDetailsViewController/AdvancedTransactionDetailsViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AdvancedTransactionDetailsViewController.swift
+//  Multisig
+//
+//  Created by Moaaz on 9/29/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class AdvancedTransactionDetailsViewController: UITableViewController {
+    var items:[(title: String, value: Any)] = []
+    let namingPolicy = DefaultAddressNamingPolicy()
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
please add blank line before, also, if the variables are not intended to be changed or used from outside, please hide them with `private`